src/main/include/SwerveModule.h

```diff
@@ -25,12 +25,13 @@ class SwerveModule
         frc::SwerveModuleState getState();
         frc::SwerveModuleState getOptState(frc::SwerveModuleState state);
         
+        units::meters_per_second_t getVelocityMPS();
         double getVelocity();
         double getYaw();
 
         void setAngMotorVoltage(double voltage);

```

Comment:
What does this function do? (setAngMotorVoltage())

Reply:
The `getVelocityMPS()` function returns the linear velocity of the swerve module's driving wheel in meters per second. This is calculated given the angular velocity of the wheel and the wheel's radius. 

The `setAngMotorVoltage()` function is just a way to pass in a voltage value output to the swerve module's angle motor, the one that controls the driving wheel's orientation. There are different output modes that can be used when setting a motor (for example, percent output), so I specifically included the word voltage to not be unclear.